frontend/BoardGUI.py

```diff
@@ -1,5 +1,5 @@
 import pygame as pg
-# from backend.Board import Board
+from backend.Board import Board
```

Comment:
I think it'd be better to keep the Board class out of this and just use the GameManager class to call all backend classes and functions.

If you're looking to use the board in the tests, we can make up a function to print board or check its status or something. I think board_gui.backend_game.board is also equivalent.

As it is, the import in line 2 is only being used to create a board in line 15, which never gets used.

Reply:
may be i can remove that particular test which uses that, and that can be refactored once we are done with game logic